ee/candi/cloud-providers/openstack/openapi/cloud_provider_discovery_data.yaml

```diff
@@ -102,6 +102,5 @@ apiVersions:
         items:
           type: string
           minLength: 1
-        minItems: 1
```

Comment:
Is it okay to have empty zones in the cloud? I always though that in OpenStack there is always at least one zone.